src/MegaPoker.t.sol

```diff
@@ -349,12 +340,54 @@ contract MegaPokerTest is DSTest, PokingAddresses {
         (, mat) = SpotLike(spotter).ilks("WSTETH-A");
         (,, spot,,) = VatLike(vat).ilks("WSTETH-A");
         assertEq(spot, rdiv(value, mat));
+
+        // These collateral types should not be updated after 1 hour
+        (, mat) = SpotLike(spotter).ilks("BAT-A");
+        (,, spot,,) = VatLike(vat).ilks("BAT-A");
+        assertTrue(spot != rdiv(value, mat));
+        (, mat) = SpotLike(spotter).ilks("LINK-A");
+        (,, spot,,) = VatLike(vat).ilks("LINK-A");
+        assertTrue(spot != rdiv(value, mat));
+        (, mat) = SpotLike(spotter).ilks("ZRX-A");
+        (,, spot,,) = VatLike(vat).ilks("ZRX-A");
+        assertTrue(spot != rdiv(value, mat));
+        (, mat) = SpotLike(spotter).ilks("LRC-A");
+        (,, spot,,) = VatLike(vat).ilks("LRC-A");
+        assertTrue(spot != rdiv(value, mat));
         (, mat) = SpotLike(spotter).ilks("GUNIV3DAIUSDC1-A");
         (,, spot,,) = VatLike(vat).ilks("GUNIV3DAIUSDC1-A");
-        assertEq(spot, rdiv(value, mat));
+        assertTrue(spot != rdiv(value, mat));
         (, mat) = SpotLike(spotter).ilks("WBTC-B");
         (,, spot,,) = VatLike(vat).ilks("WBTC-B");
         assertEq(spot, rdiv(value, mat));
```

Comment:
just noticed, WBTC-B test block should go in the previous set after WSTETH-A.

Reply:
I can open a separate pr eventually, as this is related to #26

Reply:
merge issue. fixed.